modules/preview-react/menu/stories/Menu.stories.mdx

```diff
@@ -1,18 +1,19 @@
 import {Meta} from '@storybook/addon-docs';
 import {Specifications} from '@workday/canvas-kit-docs';
-import {Menu, MenuItem} from '@workday/canvas-kit-preview-react/menu';
+import {DeprecatedMenu, DeprecatedMenuItem} from '@workday/canvas-kit-preview-react/menu';
 
 import {Basic} from './examples/Basic';
 import {ContextMenu} from './examples/ContextMenu';
 import {CustomMenuItem} from './examples/CustomMenuItem';
 import {Icons} from './examples/Icons';
 import {ManyItems} from './examples/ManyItems';
 
-<Meta title="Preview/Menu/React" component={Menu} />
+<Meta title="Preview/Menu/React" component={DeprecatedMenu} />
 
-# Canvas Kit Menu
+# Canvas Kit Menu (deprecated)
```

Comment:
Should we also update the Readme.md https://github.com/Workday/canvas-kit/tree/master/modules/preview-react/menu

Reply:
I don't think we need it in README file.